packages/core/src/Site/minimalLayout.njk

```diff
@@ -0,0 +1,14 @@
+<header>
```

Comment:
Do we still need `packages\core\template\minimal\_markbind\layouts\default.md`? (and the default one)

Reply:
No we don't need it anymore, I'll delete it now that we are okay with how each template `.njk` looks in comparison

Reply:
Sorry, I need to keep `packages\core\template\minimal\_markbind\layouts\default.md` to make sure `git` tracks the empty `layouts` folder. I couldn't get git to track the empty folder otherwise.

I made `default.md` empty with a message to edit the `.njk` file instead.